src/app/series/series.component.ts

```diff
@@ -4,10 +4,11 @@ import { Observable } from 'rxjs/Observable';
 import { Subject } from 'rxjs/Subject';
 
 import { CmsService } from '../core';
-import { ModalService, ToastrService } from 'ngx-prx-styleguide';
+import { ModalService, TabService, ToastrService } from 'ngx-prx-styleguide';
 import { SeriesModel } from '../shared';
 
 @Component({
+  providers: [TabService],
```

Comment:
I did put a ticket in the styleguide to document this so it doesn't trip us up down the road: https://github.com/PRX/styleguide.prx.org/issues/56